components/KiboProductActions.vue

```diff
@@ -10,7 +10,9 @@
           class="sf-add-to-cart__select-quantity quantity-input"
           @input="$emit('input', $event)"
         />
-        <div class="quantity-left">{{ quantityLeft }} item(s) left</div>
+        <div v-show="isValidForAddToCart" class="quantity-left">
+          {{ quantityLeft }} item(s) left
```

Comment:
create localization for the whole string. use pluralization as well.

Reply:
Done